cogs/timer.py

```diff
@@ -0,0 +1,193 @@
+            return embed
+
+        embed = create_embed()
+        msg = await ctx.send(embed=embed, components=[button_row])
+        await make_sound('boxingbell.mp3')
+
+        on_click = msg.create_click_listener()      # ClickListener für die Buttons
+
+        @on_click.matching_id("beenden")
+        async def on_beenden_button(inter):
+            nonlocal angemeldet
+            if inter.author in angemeldet:
+                nonlocal status
+                status = ["Beendet", 0]
+                button_row.disable_buttons()
+                await ping_users()
+                await make_sound('applause.mp3')
+                angemeldet = []
+                embed = create_embed()
+                await inter.reply(embed=embed, components=[button_row], type=7)
+                on_click.kill()
+            else:
+                # Reply with a hidden message
+                await inter.reply("Nur angemeldete Personen können den Timer beenden.", ephemeral=True)
+
+        @on_click.matching_id("neustart")
+        async def on_neustart_button(inter):
+            if inter.author in angemeldet:
+                nonlocal status
+                status = ["lernen", lt]
+                embed = create_embed()
+                await inter.reply(embed=embed, components=[button_row], type=7)
+                await make_sound('boxingbell.mp3')
+                await ping_users()
+            else:
+                # Reply with a hidden message
+                await inter.reply("Nur angemeldete Personen können den Timer neu starten.", ephemeral=True)
+
+        @on_click.matching_id("skip")
+        async def on_skip_button(inter):
+            nonlocal status
+            if inter.author in angemeldet:
+                await switch_phase()
+                embed = create_embed()
+                await inter.reply(embed=embed, components=[button_row], type=7)
+            else:
+                # Reply with a hidden message
+                await inter.reply("Nur angemeldete Personen können den Timer bedienen.", ephemeral=True)
+
+        @on_click.matching_id("anmelden")
+        async def on_anmelden_button(inter):
+            if inter.author not in angemeldet:
+                angemeldet.append(inter.author)
+            embed = create_embed()
+            await inter.reply(embed=embed, components=[button_row], type=7)
+
+        @on_click.matching_id("abmelden")
+        async def on_abmelden_button(inter):
+            if inter.author in angemeldet:
+                if len(angemeldet) == 1:
+                    await on_beenden_button(inter)
+                    return
+                else:
+                    angemeldet.remove(inter.author)
+            embed = create_embed()
+            await inter.reply(embed=embed, components=[button_row], type=7)
+
+        async def decrease_remaining_time():
+            if status[1] > 1:
```

Comment:
```suggestion
            if status[1] > 2:
```
Dann sollte bei 2 Minuten nochmal -1 ausgeführt und es steht noch eine Minute lang "1 Minuten"  unter Verbliebende Zeit. Momentan steht da noch für eine Minute "0 Minuten"

Reply:
ich meine, das habe ich mit einem meiner letzten commits gelöst, evt während du getestet hast? vorher war es bei `if status[1] > 0` und hat da bei 0 noch ne Minute gehangen, habe es dann auch 1 gesetzt, bei mir springt er beim Wechsel von 1 auf 0 jetzt in die nächste Phase